### PR TITLE
F39 add iso build scripts to templates

### PR DESCRIPTION
**Pull request by jkonecny12**
Change the ISO build to take branched Fedora sources instead of Rawhide.

The Live ISO builds will be broken until fedora-39-boot iso is added to Cockpit/bots repository.

### CODE REVIEW

File: dockerfile/anaconda-live-iso-creator/lmc-build [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# ======================================
+# WARNING!
+# THIS FILE IS GENERATED FROM A TEMPLATE
+# DO NOT EDIT THIS FILE MANUALLY!
+# ======================================
+# The template is located in: lmc-build.j2
+
 #!/bin/bash
```
Comment (VladimirSlavik) [inline dockerfile/anaconda-live-iso-creator/lmc-build:8]: This will not work, I think. The hashbang needs to be special-cased in the rendering script :-/

Comment (VladimirSlavik) [reply dockerfile/anaconda-live-iso-creator/lmc-build:8]: See https://github.com/rhinstaller/anaconda/pull/5104

Comment (VladimirSlavik) [reply dockerfile/anaconda-live-iso-creator/lmc-build:8]: Shebang handling ported to f39 in #5108 - feel free to ack+merge and rebase this on top (or not if it's wrong!).

**Comment by VladimirSlavik** [comment]
Two prerequisites:
- This must be rebased on top of merged #5108.
- The fix for shellcheck action must be here, too, because that runs from the branch.

**Review by VladimirSlavik — APPROVED** [review]
Aside from the technical requirements above, LGTM, thank you!

**Comment by jkonecny12** [comment]
**UPDATED:**
* Rebased on top of the templating changes (with updated rendered scripts)

**Review by VladimirSlavik — APPROVED** [review]

**Pull request merged**